genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
```

Comment:
Great job implementing this class it works great but we can improve it a bit.

the id is not necessary here because the id is an inherited property that comes from the Item class.
I would recommend to remove this line